### DISCUSSION TITLE
tutorials/lift-off-part4/04-updating-our-trackapi-data-source

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#67] [course feedback]
# tutorials/lift-off-part4/04-updating-our-trackapi-data-source

https://www.apollographql.com/tutorials/lift-off-part4/04-updating-our-trackapi-data-source

**Comment by tvadiep** [comment]
Just a side-question, which tool/library is used to develop a docs site for API like: https://odyssey-lift-off-rest-api.herokuapp.com/docs/

**Reply by mabuyo** [reply]
Hey @tvadiep - we used the `swagger-ui-express` package: https://www.npmjs.com/package/swagger-ui-express

**Reply by tvadiep** [reply]
Thank you a lot for letting me know.
